src/components/asset-list/RecyclerAssetList2/core/useMemoBriefSectionData.ts

```diff
@@ -41,6 +41,10 @@ export default function useMemoBriefSectionData() {
           return false;
         }
 
+        if (data.type === CellType.NFTS_HEADER && !arr[arrIndex + 2]) {
+          return false;
+        }
+
         if (data.type === CellType.NFTS_HEADER && !arr[arrIndex + 1]) {
```

Comment:
can u delete these lines as they were supposed to do the same thang

Reply:
OH forgot the old PR actually got merged into develop. yes